source/standards/hosting.html.md.erb

```diff
@@ -20,6 +20,14 @@ We have assessed our choice of cloud platforms to make sure they:
 
 See the [Service Manual](https://www.gov.uk/service-manual/technology/deciding-how-to-host-your-service) for more information on how to host your service.
 
+## Hosting Serverless Solutions
+
+With the growth and maturity of Function as a Service (FaaS) solutions such as [AWS Lambda](https://aws.amazon.com/lambda/) and associated technologies, GDS is developing ways of working to provide Serverless Solutions running on [Amazon Web Services (AWS)](https://aws.amazon.com). 
+
+Serverless Solutions allow GDS to hand off many responsibilities such as availability of compute capacity (servers), scaling and many complex security issues, to AWS, enabling teams to focus more on their service rather than on the underlying infrastructure.  We expect this to reduce both the per transaction cost and Total Cost of Ownership (TCO) over the lifetime of a service.
+
```

Comment:
```suggestion

```
I don't think we need this paragraph. Other than that, this looks great 👍 

We can add more serverless content in future PRs